readthedocs/core/middleware.py

```diff
@@ -203,11 +203,13 @@ class FooterNoSessionMiddleware(SessionMiddleware):
     """
 
     def process_request(self, request):
-        if ('api/v2/footer_html' in request.path_info and
-                settings.SESSION_COOKIE_NAME not in request.COOKIES):
-            # Hack request.session otherwise the Authentication middleware complains.
-            request.session = {}
-            return
+        ignore_urls = ['api/v2/footer_html', '/sustainability/view/', '/sustainability/click/']
+        for url in ignore_urls:
+            if (url in request.path_info and
```

Comment:
might be best to make this a `starts_with` instead? the probability of someone having a path that matched '.*/sustainability/view/.*' is very low, but could result in some very wonky behavior on private docs on the .com, where sessions are used in viewing docs. If some docs had a pattern that matched, they would be asked to authenticate, or who knows what.